distribution/docs/pom.xml

```diff
@@ -44,117 +44,128 @@
-                                    <outputDirectory>${project.build.directory}/apps/ui</outputDirectory>
-                                </artifactItem>
-                            </artifactItems>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
         </plugins>
     </build>
-
+    <profiles>
+        <profile>
+            <id>documentation</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>1.5</version>
+                        <executions>
+                            <execution>
+                                <id>attach-omnibus</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>attach-artifact</goal>
+                                </goals>
+                                <configuration>
+                                    <artifacts>
+                                        <artifact>
+                                            <file>${project.build.directory}/docs/html/DDFOmnibus.html</file>
+                                            <type>html</type>
+                                            <classifier>DDFOmnibus</classifier>
+                                        </artifact>
+                                        <artifact>
+                                            <file>${project.build.directory}/docs/pdf/DDFOmnibus.pdf</file>
+                                            <type>pdf</type>
+                                            <classifier>DDFOmnibus</classifier>
+                                        </artifact>
+                                    </artifacts>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <version>2.10</version>
+                        <executions>
+                            <execution>
+                                <id>unpack</id>
+                                <phase>generate-resources</phase>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <configuration>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>ddf.admin</groupId>
```

Comment:
All of these versions can be either catalog or platform. We probably shouldn't have separate versions for each of these apps since really only catalog and platform matter at this point.